### DISCUSSION TITLE
How can I send email with SMTP and SSL/TSL

### DISCUSSION THREAD

**Discussion by mdobydullah** [postalserver/postal#1510] [Help with using Postal] [answered]
Hello.

Currently, I can send email with SMTP (port 25). I tested with https://www.smtper.net. It works. Now I want to send email with SSL/TSL (465 or 587). How can I do it?

I've installed SSL and changed port 25 to 587/465 in postal config file but doesn't work. Mail server fails to send if I tick "Use Secured Connection" on https://www.smtper.net. 

From postal admin panel, if I send email from "send message" page, it sends with SSL.

Thank you.

**Comment by mdobydullah** [comment, marked as answer]
I got the solution. You can check these issues:

https://github.com/postalserver/postal/discussions/1512#discussioncomment-1092616

https://github.com/postalserver/postal/issues/1511

Thanks.

**Reply by hswamy** [reply]
Hello @mdobydullah , Can you please share how did you fix the issue. I do have similar problem.

Any help much appreciated

**Reply by solracsf** [reply]
Yes, don't use the forums to only ask for help, but also to provide solutions and help others... :(

**Reply by mdobydullah** [reply]
Hi @hswamy,

Sorry for late reply. I was out of work. I didn't check GitHub during vacation.

Please have a look at these issues:

https://github.com/postalserver/postal/discussions/1512#discussioncomment-1092616

https://github.com/postalserver/postal/issues/1511

**Reply by mdobydullah** [reply]
Hi @acsfer,

Yeah sure. We should also wait for response. We don't know the current situation of each person.

We can also use issue search option. Sometimes we get solution from other issues.

Thanks.

**Reply by hswamy** [reply]
Thanks for the response @mdobydullah , I went through your issue solution and I checked the smtp logs and i see permission denied as below.

![image](https://user-images.githubusercontent.com/43389515/131549870-edabf014-b4d9-482e-b395-edb41ea70465.png)


Any idea ?

**Reply by hswamy** [reply]
I gave read permission to .key and .crt file,

But still if i use -UseSSL, i get an error saying Send-MailMessage : The remote certificate is invalid according to the validation procedure

**Reply by mdobydullah** [reply]
Sorry @hswamy. I don't have any idea. You can take a look at [their docs](https://docs.postalserver.io/).
